app/src/main/java/com/kabouzeid/gramophone/ui/activities/AlbumDetailActivity.java

```diff
@@ -327,6 +388,7 @@ public void setStatusbarColor(int color) {
     private void setAlbum(Album album) {
         this.album = album;
         loadAlbumCover();
+        loadWiki();
```

Comment:
So I should add an option in settings to allow the users to set auto download of wiki and bio, as made for album artworks ?